src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,172 @@
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-photos"
+  alt="Photos from the original Milgram experiment, credit Stanley Milgram."
+  caption="Photos from the original Milgram experiment, credit Stanley Milgram"
+/>
+
+### What would you do?
+
+Let's take a minute to pause here - put yourself in the shoes of the teacher. You're in a lab, and you're told that you're taking part in a study on learning and memory. Everything you're seeing and hearing seems to be controlled, official, and legititmate.
+
+Do you think you would continue to administer shocks to the learner, even when they were clearly in pain?
+
+What do you think the average person would do?
+
+<br />
+<Colophon />
+
+### The Experiment: Results
+
+As you might expect, every participant in the study indicated a desire to stop the experiment at least once.
+
+Before conducting the experiment, researchers surveyed a cohort of students and fellow researchers, to get a sense of what they thought the average person would do. The average guess was that 1.2% of people would continue the experiment through the full 450V shock.
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/predicted-results"
+  alt="Predicted Results: 1.8% complete the full series of shocks, 98.2% refuse to continue at some point"
+  caption="Surveys predicted that very few participants would carry out the full series of shocks"
+/>
+
+<br />
+
+However, the actual results of the experiment revealed a surprising truth about human behavior:
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/actual-results"
+  alt="Actual Results: 59.1% completed the full series of shocks, 40.9% refused to continue at some point"
+  caption="The experimental results were much more grim."
+/>
+
+Recreations of this experiment have produced similar results, [as recently as 2009](https://www.apa.org/news/press/releases/2008/12/replicating-milgram).
+
+## Obedience and the Perils of Twitter
+
+I can't help but draw a connection between the Milgram experiment and the current state of Twitter. The experiment was designed to test the willingness of people to obey authority figures. Twitter's recent acquisition by a megalomaniacal celebrity narcissist is an amplified example of this phenomenon in action.
+
+Under Musk's leadership, Twitter's employee base has been slashed from 7,500 to approximately 2,000 in the last month, in an attempt to cut costs after a purchase that only made sense for a rich, vein billionaire who wanted to flex his ego.
+
+I've seen plenty of discussion online that many of twitter's remaining engineers are likely tied to H1B visas, and are unable to leave their job without risking deportation. There is no doubt another contingent of employees who believe in Musk's vision, who are bought into the idea of being an elite engineer working on a desperately slim team, and are willing to work for him despite the fact that he's a complete and utter asshole.
```

Comment:
The H1B piece of this has been largely disputed fwiw.

Instead of "who are bought into" would change to just "are bought into"

The "complete and utter asshole" line is pure opinion and may discredit the argument you're trying to make. Would maybe tie it to his "complete disregard for moderation best practices and repeatedly lying about his account review policies."